.github/workflows/dapr.yml

```diff
@@ -173,3 +180,33 @@ jobs:
             --body "${RELEASE_BODY}" \
             --prerelease true \
             ${RELEASE_ARTIFACT[*]}
+      - name: Package Helm chart
+        env:
+              HELM_CHARTS_DIR: charts/dapr
+        run: |
+          if [ "$LATEST_RELEASE" = "true" ]; then
+            mkdir -p ${DAPR_REPO_CODE_PATH}/${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}
+            helm package ${DAPR_REPO_CODE_PATH}/${{ env.HELM_CHARTS_DIR }} --destination ${DAPR_REPO_CODE_PATH}/${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}
+          fi
+      - name: Checkout Helm Charts Repo
+        uses: actions/checkout@v2
+        env:
+              DAPR_HELM_REPO: dapr/dapr-helm-charts.io
+        with:
+          repository: ${{ env.DAPR_HELM_REPO }}
+          ref: refs/heads/master
+          token: ${{ secrets.DAPR_BOT_HELM_TOKEN }}
+          path: ${{ env.DAPR_HELM_REPO_CODE_PATH }}
+      - name: Upload helm charts to Helm Repo
+        run: |
+          if [ "$LATEST_RELEASE" = "true" ]; then
+            cp ./${{ env.DAPR_REPO_CODE_PATH }}/${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}/* ./${{ env.DAPR_HELM_REPO_CODE_PATH }}/
+            cd ./${{ env.DAPR_HELM_REPO_CODE_PATH }}
+            git config user.name "GitHub Actions Bot"
+            git config user.email "<>"
+            git add --all
+            git commit -m "New Helm Chart change - ${{ env.DAPR_VERSION }}"  
+            git push
+            # Cleanup Helm package dir
+            rm -rf ${{ env.DAPR_REPO_CODE_PATH }}/${{ env.ARTIFACT_DIR }}/${{ env.HELM_PACKAGE_DIR }}
```

Comment:
we do not need additional clean-up step because vm will be reverted after this build is done.